tests/test/services/services_test.go

```diff
@@ -76,6 +102,62 @@ var _ = Describe("Services", func() {
 					fmt.Sprintf("the %s HelmRelease references a HelmRepository (%s) that doesn't exist in the %s directory",
 						helmRelease.Name, helmRelease.Spec.Chart.Spec.SourceRef.Name, HelmRepoDirectory))
 			})
+
+			It("should reference a ConfigMap that exists in default directory", func() {
+				configMapRefs := helmRelease.Spec.ValuesFrom
+				chartVer := helmRelease.Spec.Chart.Spec.Version
+				Expect(len(helmRelease.Spec.ValuesFrom)).To(BeNumerically(">", 0))
+				defaultMapFound := false
+				for _, configMapRef := range configMapRefs {
+					if configMapRef.Kind == ConfigMapKind {
+						Expect(configMapRef.Name).To(ContainSubstring(chartVer))
+						serviceName := helmRelease.ObjectMeta.Name
+						configMapFilePath := path.Join(ServicesDirectory, serviceName, chartVer, DefaultDirectoryName, DefaultConfigMapFileName)
+						Expect(configMapFilePath).To(BeAnExistingFile())
+						configMap, err := files.GetConfigMapObjectFromFile(configMapFilePath)
+						Expect(err).ShouldNot(HaveOccurred())
+						Expect(configMapRef.Name).Should(Equal(configMap.ObjectMeta.Name))
+						defaultMapFound = true
+						break
+					}
+				}
+				Expect(defaultMapFound).To(Equal(true))
```

Comment:
💭 do we need to check that bool for anything?

I think all the above assertions validate that the ConfigMap exists. aka could there ever be a case where this assertion would fail but the above would pass?

Reply:
If all the refs in ValuesFrom doesn't point to a ConfigMap kind then the for loop is bypassed and we should fail as well.

But now I look at this again, I better change the var name `configMapRefs` to .. e.g., `valuesFrom`